src/server/plugins/engine/views/file-upload.html

```diff
@@ -1,8 +1,5 @@
 {% extends 'index.html' %}
 
-{% from "partials/summary-detail.html" import summaryDetail %}
-{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
-{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
```

Comment:
Have you found a linter/VS code plugin for this or did you just check manually?

Reply:
Nah just force of habit every time I'm in a Nunjucks file, checking all the imports